docs/index.html

```diff
@@ -3,62 +3,625 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Bott Architecture</title>
+    <title>Bott - Documentation</title>
     <style>
+        * {
```

Comment:
Let's create a "style.css"

And do a _full_ css reset e.g. `* { all: initial; }`